tools/floodbox/floodgate/floodgate.js

```diff
@@ -507,36 +539,52 @@ export default class MiloFloodgate extends LitElement {
 
   renderCopyPreviewInfo() {
     return html`
-      <div class="preview-info info-box">
+      <div class="tab-step" data-id="preview">
         <h3>Preview Copied Files</h3>
         <p>Previewing copied files... </p>
-        <p>Files previewed: ${this._previewedFilesCount} | Preview errors: ${this._previewErrorList.length}</p>
+        <div class="detail-cards copy-preview-cards">
+          ${floodbox.renderBadge('Remaining', this._copiedFilesCount - this._previewedFilesCount)}
+          ${floodbox.renderBadge('Preview Errors', this._previewErrorList.length, true)}
+          ${floodbox.renderBadge('Success', this._previewedFilesCount)}
+          ${floodbox.renderBadge('Total', this._previewedFilesCount + this._previewErrorList.length)}
+        </div>
+        <div class="detail-lists">
+          ${floodbox.renderList('Preview Errors', this._previewErrorList)}
+        </div>
         <p>Duration: ~${this._previewDuration} seconds</p>
       </div>
-      ${this._copiedToPink ? this.renderDone() : nothing}
     `;
   }
 
   renderCopyInfo() {
     return html`
-      <div class="copy-info info-box">
+      <div class="tab-step" data-id="copy">
         <h3>Copy Content to Pink Site</h3>
         <p>Copying files to "${this._pinkSitePath}"... </p>
-        <p>Files to copy: ${this._filesToCopy.length} | Files copied: ${this._copiedFilesCount}</p>
-        <p>Copy errors: ${this._copiedErrorCount}</p>
+        <div class="detail-cards copy-cards">
+          ${floodbox.renderBadge('Remaining', this._filesToCopy.length - this._copiedFilesCount - this._copiedErrorList.length)}
+          ${floodbox.renderBadge('Copy Errors', this._copiedErrorList.length, true)}
+          ${floodbox.renderBadge('Success', this._copiedFilesCount)}
+          ${floodbox.renderBadge('Total', this._copiedFilesCount + this._copiedErrorList.length)}
+        </div>
+        <div class="detail-lists">
+          ${floodbox.renderList('Copy Errors', this._copiedErrorList)}
+        </div>
         <p class="${this._copyDuration === 0 ? 'hide' : ''}">Duration: ~${this._copyDuration} seconds</p>
       </div>
-      ${this._startPreview ? this.renderCopyPreviewInfo() : nothing}
     `;
   }
 
   renderReferencedFragmentsAssetsInfo() {
     return html`
-      <div class="fragments-assets-info info-box">
-        <h3>Find Referenced Fragments and Assets</h3>
-        <p>Found ${this._fragmentsAssets.size} referenced fragments and assets.</p>
+      <div class="tab-step active" data-id="find">
+        <h3>Find Files, Referenced Fragments and Assets</h3>
+        <div class="detail-cards find-cards">
+          ${floodbox.renderBadge('Pages', this._filesToCopy.length)}
+          ${floodbox.renderBadge('Fragments', this._fragmentsAssets.size)}
```

Comment:
```suggestion
          ${floodbox.renderBadge('Fragments & Assets', this._fragmentsAssets.size)}
```

Reply:
the & would be problematic as the name is converted as part of a classname elsewhere. My suggestion, given the details I mentioned in my slack to you would be to have a separate card for fragments and one for assets we would then need to setup the url object to include whether it was a fragment or an asset in addition to the other details I mentioned.